frontend/src/pages/FormBCSCPage.vue

```diff
@@ -332,6 +332,18 @@ watch([error], () => {
   );
   setScrollPoint("top-notification");
 });
+
+const { error:validationError } = useFetch(`/api/clients/individual/${ForestClientUserSession.user?.userId.replace('bcsc\\','')}?lastName=${ForestClientUserSession.user?.lastName}`);
+watch([validationError], () => {
+  if (validationError.value.response?.status === 409) {
+    updateValidState(3, false);
```

Comment:
Why the index here is 3?
I believe each index should be reserved to each possible additional contact.

Reply:
True, I'll fix it right now